include/cinder/Path2d.h

```diff
@@ -109,8 +109,27 @@ class Path2d {
 	//! Returns the precise bounding box around the curve itself. Slower to calculate than calcBoundingBox().
 	Rectf	calcPreciseBoundingBox() const;	
 
-	//! Returns whether the point \a pt is contained within the boundaries of the path
-	bool	contains( const vec2 &pt ) const;
+	//! Returns whether the point \a pt is contained within the boundaries of the Path2d. If \a evenOddFill is \c true (the default) then Even-Odd fill rule is used, otherwise, the Winding fill rule is applied.
+	bool	contains( const vec2 &pt, bool evenOddFill = true ) const;
+
+	//! Returns the minimum distance from point \a pt to the Path2d
+	float	calcDistance( const vec2 &pt ) const;
+	//! Returns the minimum distance from point \a pt to segment \a segment
+	float	calcDistance( const vec2 &pt, size_t segment ) const { return calcDistance( pt, segment, 0 ); }
+	//! Returns the minimum distance from the Shape2d to point \a pt. For points inside the Shape2d, the distance is negative.
+	float	calcSignedDistance( const vec2 &pt ) const
```

Comment:
I think moving these type of methods, and the similar one in `Shape2d`, would clean up the interface a tad, which is already quite a bit to take in as they are fairly complex classes and there isn't much to be gained by in-lining them.